lib/resty/healthcheck.lua

```diff
@@ -1162,6 +1178,7 @@ end
 -- * `checks.active.unhealthy.tcp_failures`: number of TCP failures to consider a target unhealthy
 -- * `checks.active.unhealthy.timeouts`: number of timeouts to consider a target unhealthy
 -- * `checks.active.unhealthy.http_failures`: number of HTTP failures to consider a target unhealthy
+-- * `checks.passive.type`: "http", "https" or "tcp" (default is "http")
```

Comment:
there is no distinction between http and https for passive checks, or am I missing something? For active there it is based on the probe we're sending, but not for passive? not sure though

Reply:
there isn't, I added it here for symmetry, since users will most likely enter the same value in both.